stories/components/navbar/NavLink.stories.js

```diff
@@ -0,0 +1,7 @@
+import NavLink from '../../components/navbar/NavLink.js'
+
+export default {
+    component: NavLink,
+};
+
+export const Basic = <NavLink/>
```

Comment:
This component has 3 required parameters `{ path, item, mode }`, please see the component for more details